### ISSUE TITLE
Select which themes to show

### ISSUE THREAD

**Issue by SukcesStrony** [envato/envato-wordpress-toolkit#123]
Hi,

A lot of companies (including mine) buy themes for their clients on ThemeForest. Currently when we use the toolkit, all themes are seen by all clients, which is not good, as 1 client usually should have access to only 1 theme. It would be great if the toolkit had a simple field to put in the license number (or some other ID) to limit the number of displayed themes available for installing and updating to only 1. I kindly request such a feature.

If this is not the correct place for this request, please point me to one.

All the best,
Chris

**Comment by velarde** [comment]
Yes, Would also like this feature to happen. Don't want all themes to show up by default.
